src/zenodo_client/struct.py

```diff
@@ -121,6 +121,9 @@ class Metadata(BaseModel):
     license: Optional[str] = "CC0-1.0"
     publication_type: Optional[PublicationType] = None
     image_type: Optional[ImageType] = None
+    communities: Optional[list] = []
```

Comment:
You mentioned in the issue that communities are objects with an `id` key. This doesn't appear to reflect that

Reply:
See updates in dd69b7a. I checked the zenodo API and actually it should have been `identifier`

Reply:
Thanks for checking - the tests suddenly failed and it was too late for me to properly track this remaining issue.